### ISSUE TITLE
Micro-stutters with some sources. 

### ISSUE THREAD

**Issue by rijnhard** [Stremio/stremio-shell#247]
Relates to what is likely an upstream bug: https://github.com/mpv-player/mpv/issues/7707

The problem is ghat right now I don't have the option to play in an external player so I can't narrow it down.

I have monitored my resource usage pretty hard and cpu is at about 30%, RAM  4.6/6.8GB and a lot of swap left.
Ive also switched to an ssd so it won't be disk.

It's also very inconsistent, some files are worse than others, and most don't have it at all.

That will be fixable with direct editing of mpv config, or maybe I'm missing something with the external player? It should be right click during playback right?

**Comment by TRtomasz** [comment]
Yes option to play in the external player is in the context menu or in the new menu which you can open by clicking great icon from the player UI (bottom right corner) Players will be displayed there if you have them installed in the standard location

**Comment by rijnhard** [comment]
Right, so this doesn't work with the flatpak version, likely due to the sandboxing.
I'll try to reinstall the deb and compare results, but right now this is DEFINITELY happening in stremio and not in the latest mpv directly (from mpv-tests ppa)

**Comment by rijnhard** [comment]
Closing this as its not an issue anymore. Was probably vaapi related.
